lib/tasks/bootstrap.rake

```diff
@@ -142,7 +145,7 @@ EOS
           {key: "RAILS_LOG_TO_STDOUT", value: "true", secret: false},
           {key: "GITHUB_ORGANIZATION", value: ENV['GITHUB_ORGANIZATION'], secret: false},
           {key: "DATABASE_URL",  value: ENV["DATABASE_URL"], secret: true},
-          {key: "SECRET_KEY_BASE", value: SecureRandom.hex(64), secret: true},
+          {key: "SECRET_KEY_BASE", value: secret_key_base, secret: true},
```

Comment:
I may be crazy, but it looks to me like this `secret_key_base` local variable is defined in the task `bcn:bootstrap`, and right here it is being used by the task `bcn:bootstrap:remote`. Those are different Ruby scopes so I suspect this line here will error out.

We can probably just keep the inline call to `SecureRandom.hex(64)` here, right?

Reply:
Oh yeah. That is odd. I wonder why this worked... It may have been because the variable was re-evaluated. You're right. It can just be the same key base here too.